design/manual-authn-and-authz/AUTHORIZATION.md

```diff
@@ -0,0 +1,46 @@
+# Manual Authorization
+
+_Note: This guide expects that Authentication was set up as describe in the [AUTHENTICATION.md](AUTHENTICATION.md) guide_
+
+## Enabling Authorization in the Kafka
+
+Authorization has to be enabled in Kafka brokers.
+To do so, the option `authorizer.class.name` needs to be set to `kafka.security.auth.SimpleAclAuthorizer`.
+It can be set in the `Kafka` resource:
+
+```
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    config:
+      authorizer.class.name: "kafka.security.auth.SimpleAclAuthorizer"
+    ...
+  zookeeper:
+    ...
+```
+
+## Managing ACL rules
+
+The ACL rules used by the default `SimpleAclAuthorizer` plugin are stored in Zookeeper.
+Since Zookeeper is not exposed to the outside, the management of the ACL rules has to be done from inside one of the zookeeper pods.
+To add or remove any ACL rules, you have to exec into the pod and call the `bin/kafka_acls.sh` utility.
+Since Zookeeper encryption is implemented using a sidecar pod, you can internally connect to the unencrypted Zookeeper port.
+This port differs in different pods.
+It is calculated using following formula: `2181 * 10 + "the number of the pod"`.
+For example:
+* pod `my-cluster-zookeeper-0` will be listening on port `21810`
+* pod `my-cluster-zookeeper-1` will be listening on port `21811`
+* etc.
+
+Once execes into the pod, the rules can be manages as described in [Kafka documentation](http://kafka.apache.org/documentation/#security_authz).
```

Comment:
This "Once" suggests you're expecting them to exec `bash -il` and run commands interactively in a bash session. But the examples below you're just calling `kafka-acls.sh` directly from the `oc exec`, so just drop the "Once execes into the pod", I think.